juanferrub/liferay-portal
# LPS-28241 Disable the file path checking in the ComboServlet during development

These changes allow using JRebel during development.

Follow these steps for testing it:
1. Check your rebel.xml configuration for web resources:

``` xml
<application xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.zeroturnaround.com" xsi:schemaLocation="http://www.zeroturnaround.com http://www.zeroturnaround.com/alderaan/rebel-2_0.xsd">

    <classpath>
        ...
    </classpath>

    <web>
        <link target="/">
            <dir name="{$liferay-dev-home}/portal/portal-web/docroot"></dir>
        </link>
    </web>

</application>
```
1. Get this PR and apply the commit (e.g. cherry-pick) in the branch your working on. 
2. If your not using portal-developer.properties in your server JAVA_OPTS, add this property to your portal-ext.properties:

combo.check.file.path=false
1. Go ahead with your work, including class and resource hot-redeployment.
2. Confirm that JRebel is working as expected
